framework/wazuh/core/cluster/dapi/dapi.py

```diff
@@ -180,6 +180,12 @@ async def distribute_function(self) -> [Dict, exception.WazuhException]:
                 raise
             self.logger.error(f"{e.message}")
             return e
+        except exception.WazuhClusterError as e:
+            e.dapi_errors = self.get_error_info(e)
+            if self.debug:
+                raise
+            self.logger.error(f"{e.message}")
+            return e
```

Comment:
Same as above. It could be added to the `WazuhInternalError` block as follows:

```python
        except exception.WazuhInternalError as e:
            e.dapi_errors = self.get_error_info(e)
            if self.debug:
                raise
            self.logger.error(f"{e.message}", exc_info=not isinstance(exception.WazuhClusterError, e))
            return e
```